python_modules/dagster/dagster/cli/dynamic_loader.py

```diff
@@ -14,6 +15,31 @@
 
 from dagster.utils import load_yaml_from_path
 
+if sys.version_info[0] >= 3:
+    import reloader
+
+    # The reloader module allows us to specify a blacklist of modules not to reload,
+    # but we want something a bit more flexible. We only want to reload the user's
+    # pipeline code, not dagster imports, installed packages, or parts of the python
+    # install. Some of these things (like numpy) actually cannot be reloaded.
+    #
+    _reload = reloader._reload
+    def conditional_reload(m, visited):
+        if "/usr/local" in m.__file__ or "site-packages" in m.__file__:
```

Comment:
why not reload when the dagster code changes? could be useful for developing the framework?

Reply:
Sure why not, I guess users == us at the moment anyway :-)